package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "gettyimages-api",
-  "version": "5.0.1",
+  "version": "0.0.0",
```

Comment:
Since `gettyimages-api` is `0.0.0` in `package.json`, it should be the same here for consistency in publishing, right?